bigbluebutton-html5/imports/ui/components/actions-bar/mute-button/component.jsx

```diff
@@ -15,43 +14,39 @@ const intlMessages = defineMessages({
   },
 });
 
-class MuteAudio extends React.Component {
-
-  render() {
-    const {
-      isInAudio,
-      isMuted,
-      callback,
-      isTalking,
-      intl,
-    } = this.props;
-
-    if (!isInAudio) return null;
-    const muteLabel = intl.formatMessage(intlMessages.muteLabel);
-    const unmuteLabel = intl.formatMessage(intlMessages.unmuteLabel);
-
-    const label = !isMuted ? muteLabel : unmuteLabel;
-    const icon = !isMuted ? 'unmute' : 'mute';
-    const tabIndex = !isInAudio ? -1 : 0;
-    let className = null;
-
-    if (isInAudio && isTalking) {
-      className = styles.circleGlow;
-    }
-
-    return (
-      <Button
-        onClick={callback}
-        label={label}
-        color={'primary'}
-        icon={icon}
-        size={'lg'}
-        circle
-        className={className}
-        tabIndex={tabIndex}
-      />
-    );
+const MuteAudio = ({
```

Comment:
Could you make it `const MuteAudio = ({isInAudio, isMuted, callback, isTalking, intl, listenOnly}) => {` ?
The length allows it, it would just look better.